tidb-cloud/changefeed-overview.md

```diff
@@ -4,13 +4,56 @@ title: Changefeed
 
 # Changefeed
 
-TiDB Cloud provides the following changefeeds to help you stream data from TiDB Cloud:
+> **Note:**
+>
+> To use the Changefeed feature, make sure that your TiDB cluster version is v6.4.0 or later and the TiKV node size is at least 8 vCPU and 16 GiB.
+>
+> Currently, tidb cloud only allows up to 10 changefeeds per cluster, regardless of kafka and mysql sink types.
+>
+> For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), the changefeed feature is unavailable.
+
+Navigate to the **Changefeed** tab of your TiDB cluster and you can see the changefeed list. In the changefeed list, you can
+
+- View the information of the created changefeed, including changefeed's id, checkpoint, status.
+- Operate the changefeed, including pause, resume, edit, and delete the changefeed.
+
+## Create Changefeed
+
+TiDB Cloud provides the following changefeeds to help you stream data from TiDB Cloud. To create a changefeed, please refer to the tutorial:
 
 - [Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md) (Beta)
 - [Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md)
 
-To learn the billing for changefeeds in TiDB Cloud, see [Changefeed billing](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md).
+## Delete a Sink
+
+1. Navigate to the **Changefeed** tab of a cluster.
+2. Click the **Action... > Delete** of the corresponding Changefeed in Changefeed list.
+
+## Pause or resume a Sink
+
+1. Navigate to the **Changefeed** tab of a cluster.
+2. Click the **Action... > Pause/Resume** of the corresponding Changefeed in Changefeed list.
+
+## Edit a MySQL sink
```

Comment:
Can we move this section to `tidb-cloud/changefeed-sink-to-mysql.md`?

Reply:
I change it to `edit a sink`